07-hosting.Rmd

```diff
@@ -373,3 +373,49 @@ You can peruse this information but keep an eye on the `restarts` count for your
 Thankfully, pm2 automatically manages the log files from your underlying processes. If you ever need to check the log files of a service, you can just run `pm2 logs run-myfile`, where `myfile` is again the name of the service obtained from `pm2 list`. This command will show you the last few lines logged from your process, and then begin streaming any incoming log lines until you exit (`Ctrl-c`). 
 
 If you want a big-picture view of the health of your server and all the pm2 services, you can run `pm2 monit` which will show you a dashboard of the RAM and CPU usage of all your services.
+
+## systemd {#systemd}
+
+(These steps are tested on Ubuntu Server 18.04 LTS)
+
+If you use a Linux server, both physical or virtual (e.g. Digital Ocean), you can use `systemd` to run Plumber as a service that can be accessed from your local network or even outside your network depending on your firewall rules.
+
+This option is a bit similar to using Docker. One of the main advantages of this procedure over using Docker is that `systemd` won't bypass firewall rules (Docker does!) and it can be less costly in terms of resources.
+
+Compared to `plumber::do_provision()` this option won't create a new droplet if you use Digital Ocean, and allows you to run your API on the same virtual machine where you have your databases and other materials. For example, you can write SQL queries from Plumber that point towards `localhost:5432` instead of going to a different IP from you API's server.
+
+To implement this option you need three steps to be run from the command line:
+
+1. Verify that you have plumber available for all users:
+
+```
+R --vanilla << EOF
```

Comment:
You can do `Rscript` or, I believe `R -e "install.packages(...`

Reply:
I checked and both are equivalent